bigtable-hbase-parent/bigtable-hbase-integration-tests-2.x/src/test/java/com/google/cloud/bigtable/hbase/IntegrationTests.java

```diff
@@ -51,8 +51,11 @@
 })
 public class IntegrationTests {
 
+  public static final int TIME_OUT_MINUTES =
```

Comment:
30, not 3.

Reply:
3, not 30.  3 minutes is a more realistic target for our integration tests.